### PR TITLE
refactor(locale): sync new locale list

### PR DESCRIPTION
**Pull request by vladikoff**
@jrgm r?

Ref: https://github.com/mozilla/fxa-auth-mailer/issues/54#issuecomment-153789199

### CODE REVIEW

**Comment by shane-tomlinson** [comment]
Super nit, the prefix `chore` is probably better than `refactor` here.

**Comment by vladikoff** [comment]
> Super nit, the prefix chore is probably better than refactor here.

I think chores do not show up in the changelog, I did this so the change will show up in the changelog in case anyone is wondering what happened to their locale. Should I leave it or change it?

**Comment by shane-tomlinson** [comment]
> I think chores do not show up in the changelog,

Can we make chores show up in the changelog?

**Comment by vladikoff** [comment]
> Can we make chores show up in the changelog?

yeap! We will need to add it there: https://github.com/vladikoff/fxa-conventional-changelog/blob/master/index.js#L28

Should we?

File: server/config/production-locales.json [hunk 1]
```diff
@@ -13,27 +14,25 @@
       "es-AR",
       "es-CL",
       "et",
-      "eu",
       "fa",
       "ff",
       "fr",
       "fy",
       "he",
-      "hi",
```
Comment (vladikoff) [inline server/config/production-locales.json]: Not here until this is fixed: https://github.com/mozilla/fxa-content-server-l10n/issues/81

**Comment by jrgm** [comment]
r+, although out of curiosity, which system, pontoon or verbatim, is the source of truth for what is being worked on. Seems like neither.

**Comment by vladikoff** [comment]
> r+, although out of curiosity, which system, pontoon or verbatim, is the source of truth for what is being worked on. Seems like neither.

We have to use both, because only certain languages move to pontoon.

**Comment by vladikoff** [comment]
Should be in sync with https://github.com/mozilla/fxa-auth-server/blob/e6617f9799567b8315575ad712c53f57fe6bd913/config/supportedLanguages.js

**Pull request merged**